### DISCUSSION TITLE
PokeMMO Halloween Update

### DISCUSSION THREAD

**Discussion by yzsvdu** [yzsvdu/RedTrainer#8] [Polls]
Event updates like Halloween often break the reflection bot due to changes in variable names, which I have to manually adjust. Once the event ends, the bot usually functions as expected again. If enough users would like an update for this Halloween event, please vote for it in the poll. Note: the event ends Nov 14

update: the devs might have added a detection mechanism to the client, because i see that the keyboard input code has changed on their side. Need more testing to figure it out so there will probably not be a fix soon until i have time to figure it out.

**Comment by GragiG** [comment]
If it is not too much work i'd really love to keep using it without a break. kodos to you

**Reply by AlvarengaLuc** [reply]
como abre o bot?
